.vscode/settings.json

```diff
@@ -10,5 +10,14 @@
         "**/node_modules": true,
         "dist/": true
     },
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "editor.formatOnSave": true,
+    "prettier.singleQuote": false,
+    "prettier.trailingComma": "es5",
+    "prettier.typescriptEnable": [
+        "typescript",
+        "typescriptreact"
+    ],
+    "prettier.semi": false,
+    "prettier.printWidth": 120
```

Comment:
Sounds good 👍  If anything ever gets out of hand its as easy as updating settings and rerunning over codebase.